src/index.html

```diff
@@ -3,11 +3,291 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>The MET</title>
+    <link
+      rel="shortcut icon"
+      href="./images/photos/THEMET.png"
+      type="image/png">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <!-- <div class="container"> -->
+    <div class="met__bar" id="menu">
+      <div class="met__nav">
+        <div class="met__logo">
+          <a href="#">
+            <img src="./images/photos/THEMET.png" alt="logo">
+          </a>
+        </div>
+        <a href="#" class="met__cancel">
+          <img src="./images/cross.svg" alt="cross">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__link">
+              Home
+            </a>
+          </li>
+
+          <li class="nav-list__item">
+            <a href="#about" class="nav-list__link">
+              About Us
+            </a>
+          </li>
+
+          <li class="nav-list__item">
+            <a href="#gallery" class="nav-list__link">
+              Gallery
+            </a>
+          </li>
+
+          <li class="nav-list__item">
+            <a href="#testimonials" class="nav-list__link">
+              Testimonials
+            </a>
+          </li>
+
+          <li class="nav-list__item">
+            <a href="#contact" class="nav-list__link">
+              Contact Me
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <div class="met__info">
```

Comment:
there are too much comments - i'll drop the issues here
![Image 2022-04-15 14 27 52](https://user-images.githubusercontent.com/39308717/163565644-adf4e7c9-78a9-4536-8b4b-eb4f3c7f7841.png)